### DISCUSSION TITLE
How to use PK-SIM to set the genotype of enzyme or transporter？

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1255] [Questions & Problems]
Dear Community,
I am working on a PBPK model I have a problem. As shown in the picture, how to set the genotype of enzyme or transporter as in the picture？
Thank you in advance :)
![微信截图_20220718154823](https://user-images.githubusercontent.com/109507330/179466722-6eca426a-a73d-4158-af11-79a369b2b9c5.jpg)

**Comment by DeniTue** [comment]
Dear @noxavier,

In the repaglinide model, the polymorphic enzymes and transporters were implemented as follows:

1) To mimic the two alleles, the polymorphic enzyme (or transporter) was implemented in the individual with "halved" reference concentration compared to the reference concentration for the enzyme/transporter from the database.
2) In the compound, two metabolism/transport processes were created per "allele" (e.g., CYP2C8 *1 and CYP2C8 *3) and respective Km and kcat values were implemented according to the values in the table.
3) In the simulation, the appropriate processes for both "alleles" were selected to match the desired geno-/phenotype.

The repaglinide model file can also be downloaded here, if you want to have a look at it yourself: https://github.com/Open-Systems-Pharmacology/Repaglinide-Model

Best,
Denise

**Reply by noxavier** [reply]
Thank you for your answer.
And I have another question, why is the value of Vmax of enzymes or transporters in compounds zero? Don't you need to add the values in the calculation parameters section?

![微信截图_20220721110504](https://user-images.githubusercontent.com/109507330/180121193-083ff3d1-c3c0-492d-a8f6-062e333150d5.jpg)

**Comment by DeniTue** [comment]
Dear @noxavier,

PK-Sim works with the kcat value, which is calculated as Vmax / Transporter concentration. When you insert values for Transporter concentration and Vmax in the upper row, kcat will be calculated by the software. If you don't insert a value for Vmax (e.g., if no information is provided in the literature), you can enter values for kcat directly. In this case, the original formula will be overwritten with this value, which is indicated by the yellow exclamation mark.

Best,
Denise

**Reply by noxavier** [reply]
That is to say, the parameter that determines the final curve is kcat. Vmax changing kcat unchanged will not change the curve?
